package/MDAnalysis/auxiliary/base.py

```diff
@@ -323,7 +323,9 @@ def __init__(self, represent_ts_as='closest', auxname=None, cutoff=None,
             self.rewind()
 
     def copy(self):
```

Comment:
in the docstring can it be made explicit if this is a deep (a completely independent thing with its own file handles etc, no shared resources) or shallow (might share a resource like a file handle with the original) copy?

Reply:
I'm going to block on this query for docs from @richardjgowers - also changelog, etc.. the usuals

Reply:
I think "deep" was meant to clarify

Reply:
Missed that, but please do take the above as an overall "docs" please - versionchanged addition would be good

Reply:
Yeah sorry I approved too hastily my brain isn't working properly today. 😅